pyerrors/input/sfcf.py

```diff
@@ -309,7 +313,7 @@ def read_sfcf_multi(path, prefix, name_list, quarks_list=['.*'], corr_type_list=
             w = specs[3]
             w2 = specs[4]
             if "files" in kwargs:
-                ls = kwargs.get("files")
```

Comment:
Was this a bug? Did the code work before when specifying the kwarg files?

Reply:
AFAIK this worked, but the thing is that the files-keyword didn't do anything in this case... With this, it would give you the option to (at least) say which ids you want, however, this is of course nto the same kind of control you have in the other cases, where you can actually say which idls you want.
For this to work, we would need to implement something similar to #185 for this read method. I coudl also take care of that, but I need some time for that I think.